python/pyiceberg/catalog/rest.py

```diff
@@ -490,6 +498,27 @@ def rename_table(self, from_identifier: Union[str, Identifier], to_identifier: U
 
         return self.load_table(to_identifier)
 
+    def alter_table(self, identifier: Union[str, Identifier], updates: Tuple[BaseTableUpdate, ...]) -> TableResponse:
```

Comment:
Shouldn't this be `update_table` rather than `alter`? Alter is typically used for metadata changes and this route is used for both.

Reply:
Works for me